aws/internal/service/ec2/actions/instance.go

```diff
@@ -0,0 +1,25 @@
+package actions
```

Comment:
Naming the package `actions` as a trial balloon.
The API documentation for many AWS services has an _Actions_ section so the named seems apt in that context.